package.json

```diff
@@ -19,18 +19,26 @@
     "build": "browserify src/all.js > dist/js/orb.js",
     "bdist": "browserify src/all.js | uglifyjs -mc > dist/js/orb.min.js",
     "deploy": "npm run bdist && npm run build && npm run version:patch && npm run push:git && npm publish",
+    "test": "./node_modules/.bin/mocha tests",
+    "test-cov": "./node_modules/.bin/istanbul cover _mocha tests",
     "version:patch": "mversion patch -m \"Release v%s\"",
     "push:git": "git push --tags origin HEAD:master"
   },
   "devDependencies": {
     "browserify": "",
+    "backbone": "",
     "express": "^4.13.0",
     "grunt": "",
     "grunt-cli": "",
     "grunt-bowercopy": "",
     "mversion": "",
+    "jquery": "",
+    "underscore": "",
     "uglify-js": "",
-    "watchify": ""
+    "watchify": "",
```

Comment:
starting to add unit tests using mocha + chai (assertion) + istanbul (code coverage)